yarn/src/main/scala/org/apache/spark/scheduler/cluster/YarnClientSchedulerBackend.scala

```diff
@@ -125,6 +125,7 @@ private[spark] class YarnClientSchedulerBackend(
    */
   private def asyncMonitorApplication(): Unit = {
     assert(client != null && appId != null, "Application has not been submitted yet!")
+    val interval = conf.getLong("spark.yarn.client.progress.pollinterval", 1000)
```

Comment:
My main concern is that we don't really have a notion of "progress" in Spark on YARN.  In MapReduce, progress traditionally referred to how far along the app was, but Spark doesn't provide a similar notion.  YARN itself does have a notion of app progress, which is not related to what's here.